README.md

```diff
@@ -29,10 +31,10 @@ A curated list of awesome open source workflow engines
 * [Titanoboa](https://titanoboa.io/) [![Stars](https://img.shields.io/github/stars/mikub/titanoboa.svg)](https://github.com/mikub/titanoboa) - Titanoboa is a platform for creating complex workflows on JVM.
 * [uTask](https://github.com/ovh/utask) [![Stars](https://img.shields.io/github/stars/ovh/utask)](https://github.com/ovh/utask/stargazers) - Automation engine that models and executes business processes declared in yaml.
 * [Wexflow](https://wexflow.github.io) [![Stars](https://img.shields.io/github/stars/aelassas/Wexflow.svg)](https://github.com/aelassas/Wexflow) - A high-performance, extensible, modular and cross-platform workflow engine.
-* [Workflow Core](https://github.com/danielgerlag/workflow-core) [![Stars](https://img.shields.io/github/stars/danielgerlag/workflow-core.svg)](https://github.com/danielgerlag/workflow-core) - Workflow Core is a light weight workflow engine targeting .NET Standard.
```

Comment:
why did you remove this ?

Reply:
duplicate with ：
![image](https://user-images.githubusercontent.com/3401462/76063698-345d1b80-5fc3-11ea-9b4a-52d5fa4e2d14.png) in catalog Full fledged product.

Reply:
@meirwah  any comments ? or  you need keep the duplicate item ?